examples/jms-consumer-acknowledgement/jms_consumer_acknowledgement.md

```diff
@@ -0,0 +1,18 @@
+# JMS message producer - Manual acknowledgment
+
+The `jms:MessageConsumer` allows manual acknowledgment for received messages. A `jms:MessageConsumer` can be initialized by using a `jms:Session` object. To configure manual acknowledgment mode,  create a `jms:Session` object with acknowledge mode set to `jms:CLIENT_ACKNOWLEDGE`.
+
+::: code jms_consumer_acknowledgement.bal :::
+
+## Prerequisites
+- Start a [ActiveMQ broker](https://activemq.apache.org/getting-started) instance.
```

Comment:
```suggestion
- Start a [ActiveMQ broker](https://activemq.apache.org/getting-started) instance.
```
We don't need a bullet point if we have only one point.

Reply:
Fixed with [29db9a5527a2a51e3c7bd84cb993a17d1558c238](https://github.com/ballerina-platform/ballerina-distribution/pull/4880/commits/29db9a5527a2a51e3c7bd84cb993a17d1558c238)